src/Proto.Persistence/InMemoryProviderState.cs

```diff
@@ -60,5 +51,15 @@ public Task PersistSnapshotAsync(string actorName, ulong eventIndex, object snap
             _snapshots[actorName] = Tuple.Create((object) snapshot, eventIndex);
             return Task.FromResult(0);
         }
+
+        public Task DeleteEventsAsync(string actorName, ulong fromEventIndex)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task DeleteSnapshotsAsync(string actorName, ulong fromEventIndex)
+        {
+            throw new NotImplementedException();

```

Comment:
I think we should remove InMemoryProvider. Dont see the use case for it really. If we need a default provider maybe we should persist om disk in a File.